packages/create-atomic/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "module": "commonjs",
+    "module": "ES2020",
```

Comment:
Can you try with `nodenext` ? would require typescript 4.5 but it should be the best way now I reckon.

https://www.typescriptlang.org/docs/handbook/esm-node.html

Reply:
You mean 4.6? Doesn't appear to work with 4.5. What's the issue with ES2020 ?